en/developer_guide/entity_list_ajax_pagination.rst

```diff
@@ -0,0 +1,45 @@
+Entity list with ajax pagination
+================================
+
+To list the entities Open Orchestra uses the  `DataTables`_ plugin.
```

Comment:
You could start with a little bit of context : 

To list the entities in an ergonomic way, Open Orchestra uses the `DataTables`_ plugin.